src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -166,6 +169,23 @@ class BaseOptionsSelector extends Component {
         return defaultIndex;
     }
 
+    updateSearchValue(value) {
+        if (value.length > this.props.maxLength) {
+            this.setState({
+                searchValue: value,
```

Comment:
@allroundexperts 
Calling `this.props.onChangeText(value)` will display the result for exceeded search value.
If we are going to allow that, we need also to remove `searchValue` because `this.props.value` it always updated when `onChange` and will used in the TextInput instead.

Reply:
updated!

Reply:
@allroundexperts 
I added `searchValue` to use it to get the length of search text, because I was stopped calling `this.props.onChangeText` after exceed, but now we will always use `this.props.onChangeText`, so I removed `searchValue` and back using `this.props.value`.
I will update it to be
```javascript
updateSearchValue(value) {
    this.setState({
        errorMessage: value.length > this.props.maxLength ? this.props.translate('common.error.characterLimitExceedCounter', {length: value.length, limit: this.props.maxLength}) : '',
    });

    this.props.onChangeText(value);
}
```